client/src/balance.rs

```diff
@@ -86,10 +82,12 @@ impl<'a, 'b> ClientCommand<'a, 'b> for GetBalance {
 
     fn run(matches: &ArgMatches<'_>) {
         let verbose = common::verbose::get(matches);
-        let node_address = common::node_address::get(matches);
         let rpc_id = common::rpc_id::get(matches);
-        let args = balance_args::get(matches);
-        let response = Self::request_with_map_params(verbose, &node_address, rpc_id, args);
+        let node_address = common::node_address::get(matches);
+        let params = balance_args::get(matches);
+        let response = casper_client::RpcCall::new(rpc_id, verbose)
```

Comment:
Let's import `casper_client::RpcCall`.

(This is applicable in a quite a few places I believe)